nuxeo-coldstorage-web/elements/nuxeo-retrieve-content-from-coldstorage-button.js

```diff
@@ -0,0 +1,141 @@
+/**
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import '@polymer/iron-icon/iron-icon.js';
+import '@polymer/iron-icons/iron-icons.js';
+import '@nuxeo/nuxeo-elements/nuxeo-connection.js';
+import '@nuxeo/nuxeo-elements/nuxeo-element.js';
+import '@nuxeo/nuxeo-elements/nuxeo-operation.js';
+import '@polymer/paper-button/paper-button.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/polymer/lib/elements/dom-if.js';
+import '@polymer/polymer/lib/elements/dom-repeat.js';
+import { FiltersBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js';
+import { FormatBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-format-behavior.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-dialog.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-tooltip.js';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-action-button-styles.js';
+
+/**
+`nuxeo-retrieve-content-from-coldstorage-button`
+@group Nuxeo UI
+@element nuxeo-retrieve-content-from-coldstorage-button
+*/
+class RetrieveFromColdStorage extends mixinBehaviors([FiltersBehavior, FormatBehavior], Nuxeo.Element) {
+  static get template() {
+    return html`
+      <style include="nuxeo-action-button-styles nuxeo-styles"></style>
+      <nuxeo-operation
+        id="opRetrieve"
+        op="Document.RequestRetrievalFromColdStorage"
+        input="[[document.uid]]"
+        params='{"numberOfDaysOfAvailability": 1}'
```

Comment:
we need to understand better what this param means and what its value should be

Reply:
@andreacornaglia could you remove the param `numberOfDaysOfAvailability` since the PR https://github.com/nuxeo/nuxeo-coldstorage/pull/47 @doulba will merge it sooner to 10.10. Andrea, at meanwhile you can rebase your branch on the Abdoul's one `task-NXP-30364-retrieve-content-with-default-value`